scripts/error-codes/codes.json

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
so did this start with the codes.json from react 12 that I built, with more errors extracted?  If not it should - the idea is that this file is append-only, so it doesn't need to be versioned for each react.